pkg/operator/target_config_reconciler.go

```diff
@@ -150,11 +150,11 @@ func generateConfigMapString(requestedStrategies []deschedulerv1beta1.Strategy)
 	// There is no need to do validation here. By the time, we reach here, validation would have already happened.
 	for _, strategy := range requestedStrategies {
 		switch strings.ToLower(strategy.Name) {
-		case "duplicates":
+		case "duplicates", "RemoveDuplicates":
 			policy.Strategies["RemoveDuplicates"] = deschedulerapi.DeschedulerStrategy{Enabled: true}
-		case "interpodantiaffinity":
+		case "interpodantiaffinity", "RemovePodsViolatingInterPodAntiAffinity":
 			policy.Strategies["RemovePodsViolatingInterPodAntiAffinity"] = deschedulerapi.DeschedulerStrategy{Enabled: true}
-		case "lownodeutilization":
+		case "lownodeutilization", "LowNodeUtilization":
```

Comment:
Why do you need this if you have `ToLower` at the top?

Reply:
updated, thanks